source/GameContainer.js

```diff
@@ -86,6 +86,7 @@ export default class GameContainer extends Pixi.Container {
         }
         this.spawnWaveInterval = 0
         this.difficulty = Reference.DIFFICULTY[0]
+        game.removeChildAt(25)
```

Comment:
Yeah, definitely never do this. Not only is it sketchy, it actually wouldn't work if you believed more than one player would have the sword at once.